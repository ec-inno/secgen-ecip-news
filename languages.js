module.exports = {
  defaultLangKey: 'en',
  // The `languages` list follows ECL spec. (splash page, language swticher, etc.)
  languages: [
    // { lang: 'bg', label: 'български' },
    // { lang: 'es', label: 'español' },
    // { lang: 'cs', label: 'čeština' },
    // { lang: 'da', label: 'dansk' },
    // { lang: 'de', label: 'Deutsch' },
    // { lang: 'et', label: 'eesti' },
    // { lang: 'el', label: 'ελληνικά' },
    { lang: 'en', label: 'English' },
    { lang: 'fr', label: 'français' },
    { lang: 'ga', label: 'Gaeilge' },
    // { lang: 'hr', label: 'hrvatski' },
    // { lang: 'it', label: 'italiano' },
    // { lang: 'lv', label: 'latviešu' },
    // { lang: 'lt', label: 'lietuvių' },
    // { lang: 'hu', label: 'magyar' },
    // { lang: 'mt', label: 'Malti' },
    // { lang: 'nl', label: 'Nederlands' },
    // { lang: 'pl', label: 'polski' },
    // { lang: 'pt', label: 'português' },
    // { lang: 'ro', label: 'română' },
    // { lang: 'sk', label: 'slovenčina' },
    // { lang: 'sl', label: 'slovenščina' },
    // { lang: 'fi', label: 'suomi' },
    // { lang: 'sv', label: 'svenska' },
  ],
};
