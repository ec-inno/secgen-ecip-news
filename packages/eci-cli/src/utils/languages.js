module.exports = [
  'bg',
  'es',
  'cs',
  'da',
  'de',
  'et',
  'el',
  'en',
  'fr',
  'ga',
  'hr',
  'it',
  'lv',
  'lt',
  'hu',
  'mt',
  'nl',
  'pl',
  'pt',
  'ro',
  'sk',
  'sl',
  'fi',
  'sv',
];
