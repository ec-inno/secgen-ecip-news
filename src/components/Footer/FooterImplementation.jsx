import React, { useContext } from 'react';

import I18nContext from '../../context/I18n';

// ECL native component.
import Footer from './Footer';

const FooterLanguage = () => {
  const { locale } = useContext(I18nContext);
  const data = require(`./data/${locale}.json`);

  return <Footer className="ecl-footer--custom" {...data} />;
};

export default FooterLanguage;
