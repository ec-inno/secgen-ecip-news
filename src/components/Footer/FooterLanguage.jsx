import React from 'react';

import useTranslations from '../../utils/useTranslations';

// ECL native component.
import Footer from './Footer';

const FooterLanguage = () => {
  const translation = useTranslations('footer');

  return <Footer className="ecl-footer--custom" {...translation} />;
};

export default FooterLanguage;
