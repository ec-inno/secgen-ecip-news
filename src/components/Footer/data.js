// Simple content for demo
const data = {
  en: {
    backToTop: {
      variant: 'standalone',
      label: 'Go to top',
      href: '#top',
      icon: {
        shape: 'ui--corner-arrow',
        size: 'fluid',
      },
    },
    sections: [
      {
        title: 'Contact the EU',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: '00 800 6 7 8 9 10 11',
                href: '#',
              },
              before: 'Call us ',
            },
            {
              link: {
                variant: 'standalone',
                label: 'telephone options',
                href: '#',
              },
              before: ' or use other ',
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'contact form',
                href: '#',
              },
              before: 'E-mail us via our ',
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'contact points',
                href: '#',
              },
              before: 'Visit our ',
              after: ' in your country',
            },
          ],
        ],
      },
      {
        title: 'Find an EU social media account',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'EU social media channels',
                href: '#',
              },
              before: 'Search for ',
            },
          ],
        ],
      },
      {
        title: 'Find a European institution',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Parliament',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Council',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Council of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Commission',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Court of Justice of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Central Bank',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Court of Auditors',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European External Action Service',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Economic and Social Committee',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Committee of the Regions',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Investment Bank',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Ombudsman',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Data Protection Supervisor',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Personnel Selection Office',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Publications Office of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Agencies',
                href: '#',
              },
            },
          ],
        ],
      },
      {
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'EU institutions',
                href: '#',
              },
              before: 'Search for ',
            },
          ],
        ],
      },
    ],
    common: {
      label: 'Last published 26/06/2018',
      links: [
        {
          variant: 'standalone',
          label: 'Language policy',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Privacy policy',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Legal notice',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Cookies',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Web accessibility',
          href: '#',
        },
      ],
    },
  },
  fr: {
    backToTop: {
      variant: 'standalone',
      label: 'Go to top',
      href: '#top',
      icon: {
        shape: 'ui--corner-arrow',
        size: 'fluid',
      },
    },
    sections: [
      {
        title: 'Contact the EU',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: '00 800 6 7 8 9 10 11',
                href: '#',
              },
              before: 'Call us ',
            },
            {
              link: {
                variant: 'standalone',
                label: 'telephone options',
                href: '#',
              },
              before: ' or use other ',
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'contact form',
                href: '#',
              },
              before: 'E-mail us via our ',
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'contact points',
                href: '#',
              },
              before: 'Visit our ',
              after: ' in your country',
            },
          ],
        ],
      },
      {
        title: 'Find an EU social media account',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'EU social media channels',
                href: '#',
              },
              before: 'Search for ',
            },
          ],
        ],
      },
      {
        title: 'Find a European institution',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Parliament',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Council',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Council of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Commission',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Court of Justice of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Central Bank',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Court of Auditors',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European External Action Service',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Economic and Social Committee',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Committee of the Regions',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Investment Bank',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Ombudsman',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Data Protection Supervisor',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Personnel Selection Office',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Publications Office of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Agencies',
                href: '#',
              },
            },
          ],
        ],
      },
      {
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'EU institutions',
                href: '#',
              },
              before: 'Search for ',
            },
          ],
        ],
      },
    ],
    common: {
      label: 'Last published 26/06/2018',
      links: [
        {
          variant: 'standalone',
          label: 'Language policy',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Privacy policy',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Legal notice',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Cookies',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Web accessibility',
          href: '#',
        },
      ],
    },
  },
  bg: {
    backToTop: {
      variant: 'standalone',
      label: 'Go to top',
      href: '#top',
      icon: {
        shape: 'ui--corner-arrow',
        size: 'fluid',
      },
    },
    sections: [
      {
        title: 'Contact the EU',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: '00 800 6 7 8 9 10 11',
                href: '#',
              },
              before: 'Call us ',
            },
            {
              link: {
                variant: 'standalone',
                label: 'telephone options',
                href: '#',
              },
              before: ' or use other ',
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'contact form',
                href: '#',
              },
              before: 'E-mail us via our ',
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'contact points',
                href: '#',
              },
              before: 'Visit our ',
              after: ' in your country',
            },
          ],
        ],
      },
      {
        title: 'Find an EU social media account',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'EU social media channels',
                href: '#',
              },
              before: 'Search for ',
            },
          ],
        ],
      },
      {
        title: 'Find a European institution',
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Parliament',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Council',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Council of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Commission',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Court of Justice of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Central Bank',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Court of Auditors',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European External Action Service',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Economic and Social Committee',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Committee of the Regions',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Investment Bank',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Ombudsman',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Data Protection Supervisor',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'European Personnel Selection Office',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Publications Office of the European Union',
                href: '#',
              },
            },
          ],
          [
            {
              link: {
                variant: 'standalone',
                label: 'Agencies',
                href: '#',
              },
            },
          ],
        ],
      },
      {
        items: [
          [
            {
              link: {
                variant: 'standalone',
                label: 'EU institutions',
                href: '#',
              },
              before: 'Search for ',
            },
          ],
        ],
      },
    ],
    common: {
      label: 'Last published 26/06/2018',
      links: [
        {
          variant: 'standalone',
          label: 'Language policy',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Privacy policy',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Legal notice',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Cookies',
          href: '#',
        },
        {
          variant: 'standalone',
          label: 'Web accessibility',
          href: '#',
        },
      ],
    },
  },
};

export default data;
