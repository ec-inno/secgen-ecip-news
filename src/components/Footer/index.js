// The ECL's Footer wrapper providing language-specific content.
export { default } from './FooterImplementation';
