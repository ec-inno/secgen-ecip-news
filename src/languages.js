module.exports = {
  // Used by gatsby-source-drupal-jsonapi-eci plugin for iterating language-specific endpoints.
  langs: ['en', 'bg', 'fr'],
  // For possible redirects. Used by gatsby-plugin-i18n.
  defaultLangKey: 'en',
  // As close as possible to https://github.com/ec-europa/europa-component-library/blob/v2-dev/src/systems/ec/specs/components/language-list/demo/data--splash.js.
  languages: [
    { lang: 'bg', label: 'български' },
    { lang: 'en', label: 'English' },
    { lang: 'fr', label: 'français' },
  ],
  map: {
    bg: 'български',
    es: 'español',
    cs: 'čeština',
    da: 'dansk',
    de: 'Deutsch',
    et: 'eesti',
    el: 'ελληνικά',
    en: 'English',
    fr: 'français',
    ga: 'Gaeilge',
    hr: 'hrvatski',
    it: 'italiano',
    lv: 'latviešu',
    lt: 'lietuvių',
    hu: 'magyar',
    mt: 'Malti',
    nl: 'Nederlands',
    pl: 'polski',
    pt: 'português',
    ro: 'română',
    sk: 'slovenčina',
    sl: 'slovenščina',
    fi: 'suomi',
    sv: 'svenska',
  },
};
