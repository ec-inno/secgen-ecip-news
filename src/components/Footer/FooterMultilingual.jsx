import React from 'react';

// ECL native component.
import Footer from './Footer';

import data from './data';
import getCurrentLanguage from '../../utils/getCurrentLanguage';

const FooterMultilingual = ({ location }) => {
  const lang = getCurrentLanguage(location);
  const footerData = data[lang];
  return <Footer {...footerData} />;
};

export default FooterMultilingual;
