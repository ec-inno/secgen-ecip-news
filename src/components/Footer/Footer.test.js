import React from 'react';
import renderer from 'react-test-renderer';

import { defaultLangKey } from '../../../languages';
import Footer from './Footer';

describe('Footer', () => {
  let data = {};

  beforeAll(() => {
    data = require(`./data/${defaultLangKey}.json`);
  });

  it('renders correctly given default language data', () => {
    const tree = renderer.create(<Footer {...data} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
