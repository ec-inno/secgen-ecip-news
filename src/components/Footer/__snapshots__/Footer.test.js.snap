// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly given default language data 1`] = `
<footer
  className="ecl-footer ecl-footer--updated ecl-footer--custom"
>
  <section
    className="ecl-footer__identity"
  >
    <div
      className="ecl-container"
    >
      <h1
        className="ecl-footer__identity-title"
      >
        European Citizens' Initiative
      </h1>
      <div
        className="ecl-row"
      >
        
        <div
          className="ecl-footer__identity-info ecl-col-12 ecl-col-md-6"
        >
          <a
            className="ecl-footer__identity-link ecl-link ecl-link--standalone"
            href="/en/communication-material"
            internal="true"
          >
            Communication material
          </a>
          <a
            className="ecl-footer__identity-link ecl-link ecl-link--standalone"
            href="/en/faq"
            internal="true"
          >
            FAQ
          </a>
          <a
            className="ecl-footer__identity-link ecl-link ecl-link--standalone"
            href="/en/sitemap"
            internal="true"
          >
            Site map
          </a>
          <a
            className="ecl-footer__identity-link ecl-link ecl-link--standalone"
            href="/en/contact"
            internal="true"
          >
            Contact
          </a>
          <a
            className="ecl-footer__identity-link ecl-link ecl-link--standalone"
            href="#"
          >
            Newsletter
          </a>
        </div>
      </div>
    </div>
  </section>
  <div
    className="ecl-footer__sections"
  >
    <div
      className="ecl-container"
    >
      <div
        className="ecl-row"
      >
        <section
          className="ecl-footer__section ecl-col-12 ecl-col-md-4"
        >
          <h1
            className="ecl-footer__section-title"
          >
            Contact the EU
          </h1>
          <ul
            className="ecl-footer__section-list"
          >
            <li
              className="ecl-footer__section-item"
            >
              <span>
                <span>
                  Call us 
                </span>
                <a
                  className="ecl-footer__section-link ecl-link ecl-link--standalone"
                  href="#"
                >
                  00 800 6 7 8 9 10 11
                </a>
              </span>
              <span>
                <span>
                   or use other 
                </span>
                <a
                  className="ecl-footer__section-link ecl-link ecl-link--standalone"
                  href="#"
                >
                  telephone options
                </a>
              </span>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <span>
                <span>
                  E-mail us via our 
                </span>
                <a
                  className="ecl-footer__section-link ecl-link ecl-link--standalone"
                  href="#"
                >
                  contact form
                </a>
              </span>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <span>
                <span>
                  Visit our 
                </span>
                <a
                  className="ecl-footer__section-link ecl-link ecl-link--standalone"
                  href="#"
                >
                  contact points
                </a>
                <span>
                   in your country
                </span>
              </span>
            </li>
          </ul>
          <h1
            className="ecl-footer__section-title"
          >
            Find an EU social media account
          </h1>
          <ul
            className="ecl-footer__section-list"
          >
            <li
              className="ecl-footer__section-item"
            >
              <span>
                <span>
                  Search for 
                </span>
                <a
                  className="ecl-footer__section-link ecl-link ecl-link--standalone"
                  href="#"
                >
                  EU social media channels
                </a>
              </span>
            </li>
          </ul>
        </section>
        <section
          className="ecl-footer__section ecl-col-12 ecl-col-md-8"
        >
          <h1
            className="ecl-footer__section-title"
          >
            Find a European institution
          </h1>
          <ul
            className="ecl-footer__section-list ecl-footer__section-list--long ecl-footer__columns"
          >
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Parliament
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Council
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                Council of the European Union
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Commission
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                Court of Justice of the European Union
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Central Bank
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Court of Auditors
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European External Action Service
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Economic and Social Committee
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Committee of the Regions
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Investment Bank
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Ombudsman
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Data Protection Supervisor
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                European Personnel Selection Office
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                Publications Office of the European Union
              </a>
            </li>
            <li
              className="ecl-footer__section-item"
            >
              <a
                className="ecl-footer__section-link ecl-link ecl-link--standalone"
                href="#"
              >
                Agencies
              </a>
            </li>
          </ul>
          <ul
            className="ecl-footer__section-list ecl-footer__section-list--short"
          >
            <li
              className="ecl-footer__section-item"
            >
              <span>
                <span>
                  Search for 
                </span>
                <a
                  className="ecl-footer__section-link ecl-link ecl-link--standalone"
                  href="#"
                >
                  EU institutions
                </a>
              </span>
            </li>
          </ul>
        </section>
      </div>
    </div>
  </div>
  <section
    className="ecl-footer__common"
  >
    <div
      className="ecl-container ecl-footer__common-container"
    >
      <span
        className="ecl-footer__common-label"
      >
        Last published 26/06/2018
      </span>
      <a
        className="ecl-footer__common-link ecl-link ecl-link--standalone"
        href="#"
      >
        Language policy
      </a>
      <a
        className="ecl-footer__common-link ecl-link ecl-link--standalone"
        href="#"
      >
        Privacy policy
      </a>
      <a
        className="ecl-footer__common-link ecl-link ecl-link--standalone"
        href="#"
      >
        Legal notice
      </a>
      <a
        className="ecl-footer__common-link ecl-link ecl-link--standalone"
        href="#"
      >
        Cookies
      </a>
      <a
        className="ecl-footer__common-link ecl-link ecl-link--standalone"
        href="#"
      >
        Web accessibility
      </a>
    </div>
  </section>
</footer>
`;
